src/MensaWidget/MWFrameTest.class.st

```diff
@@ -81,6 +81,70 @@ MWFrameTest >> testCorrectTitle [
 	self assert: self widgetFrame label = 'Mensa Widget'
 ]
 
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'tmc 6/11/2024 18:17'
+}
+MWFrameTest >> testForButton [
+
+	| bool |
+	
+	bool := true.
+	self widgetFrame submorphsDo: [:each |
+		each submorphsDo: [:each2 |
+			(each2 respondsTo: #actionSelector) ifTrue: [
+				(each2 actionSelector = #expandBoxHit) ifTrue: [bool:= false]]]].	
+	self assert: bool
+
+]
+
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'tmc 6/11/2024 18:17'
+}
+MWFrameTest >> testFullScreen [
+	
+	| bool size | 
+	
+	size := 1 @ 1.
+	bool := false.
+	self widgetFrame extent: size.
+	self widgetFrame expandToFullScreen.
+	(self widgetFrame extent = size) ifTrue: [bool := true].
+	self assert: bool
```

Comment:
kannst hier direkt self asset: (self widgetFrame extent = size) und würde das auch nochmal vor das expand to fullscreen machen, damit du sichergehst das das geklappt hat

Reply:
done